src/Sylius/Behat/Page/Shop/Error/ErrorPage.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shop\Error;
```

Comment:
@lchrusciel Cause now, it's the general error page, we can use Sylius\Behat\Page\Error namespace. What do you think?

Reply:
Good idea